File: backend/src/main/java/org/jdk/project/dto/digitaltwin/DigitalTwinActionRequest.java
package org.jdk.project.dto.digitaltwin;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DigitalTwinActionRequest {
  private String robotId;
  private Long regionId;
  private Long areaId;
  private String coordinate;
  private Long warningId;
  private Long passengerId;
  private String warningInfo;
  private String warningType;
}

File: backend/src/main/kotlin/org/jdk/project/dto/digitaltwin/DigitalTwinActionRequest.kt
package org.jdk.project.dto.digitaltwin

data class DigitalTwinActionRequest(
    var robotId: String? = null,
    var regionId: Long? = null,
    var areaId: Long? = null,
    var coordinate: String? = null,
    var warningId: Long? = null,
    var passengerId: Long? = null,
    var warningInfo: String? = null,
    var warningType: String? = null,
)
